src/NzbDrone.Core/Indexers/BroadcastheNet/BroadcastheNetSettings.cs

```diff
@@ -23,6 +25,7 @@ public BroadcastheNetSettings()
         {
             BaseUrl = "http://api.broadcasthe.net/";
             MinimumSeeders = IndexerDefaults.MINIMUM_SEEDERS;
+            SearchPriority = IndexerDefaults.SEARCH_PRIORITY;
```

Comment:
Moving this off of indexer settings and into the definition (because it applies to all indexers regardless of protocol, unlike minimum seeders), means you don't need to copy paste all this code to each indexer.